database/migrations/2019_06_24_021225_create_lessons_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateLessonsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('lessons', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->unsignedInteger('category_id');
```

Comment:
It is stated in your ERD that `category_id` and `user_id` is foreign key. Please set this column as foreign key

Reply:
Noted.

Reply:
Resolved